it/controllers/RegimeAppealsControllerISpec.scala

```diff
@@ -940,6 +935,528 @@ class RegimeAppealsControllerISpec extends IntegrationSpecCommonBase with Regime
+            |    "appealInformation": {
+            |          "reasonableExcuse": "fireandflood",
+            |          "honestyDeclaration": true,
+            |          "startDateOfEvent": "2021-04-23T00:00:00",
+            |          "statement": "This is a statement",
+            |          "lateAppeal": false
+            |    }
+            |}
+            |""".stripMargin)
+        val result = await(buildClientForRequestToApp(uri = s"/$r/appeals/submit-appeal/$it/$i?isLPP=false&penaltyNumber=123456789&correlationId=uuid-1").post(
+          jsonToSubmit
+        ))
+        result.status shouldBe OK
+      }
+
+      "call the connector and send the appeal data received in the request body - returns OK when successful for loss of staff" in {
+        setEnabledFeatureSwitches(CallAPI1808HIP)
+        mockStubResponseForAuthorisedUser
+        mockSuccessfulResponse()
+
+        val jsonToSubmit: JsValue = Json.parse(
+          """
+            |{
+            |    "sourceSystem": "MDTP",
+            |    "taxRegime": "VAT",
+            |    "customerReferenceNo": "123456789",
+            |    "dateOfAppeal": "2020-01-01T00:00:00",
+            |    "isLPP": false,
+            |    "appealSubmittedBy": "customer",
+            |    "appealInformation": {
+            |						 "reasonableExcuse": "lossOfEssentialStaff",
+            |            "honestyDeclaration": true,
+            |            "startDateOfEvent": "2021-04-23T00:00:00",
+            |						 "statement": "This is a statement",
+            |            "lateAppeal": false
+            |		}
+            |}
+            |""".stripMargin)
+        val result = await(buildClientForRequestToApp(uri = s"/$r/appeals/submit-appeal/$it/$i?isLPP=false&penaltyNumber=123456789&correlationId=uuid-1").post(
+          jsonToSubmit
+        ))
+        result.status shouldBe OK
+      }
+
+      "call the connector and send the appeal data received in the request body - returns OK when successful for technical issues" in {
+        setEnabledFeatureSwitches(CallAPI1808HIP)
+        mockStubResponseForAuthorisedUser
+        mockSuccessfulResponse()
+
+        val jsonToSubmit: JsValue = Json.parse(
+          """
+            |{
+            |    "sourceSystem": "MDTP",
+            |    "taxRegime": "VAT",
+            |    "customerReferenceNo": "123456789",
+            |    "dateOfAppeal": "2020-01-01T00:00:00",
+            |    "isLPP": false,
+            |    "appealSubmittedBy": "customer",
+            |    "appealInformation": {
+            |					 	 "reasonableExcuse": "technicalIssue",
+            |            "honestyDeclaration": true,
+            |            "startDateOfEvent": "2021-04-23T00:00:00",
+            |            "endDateOfEvent": "2021-04-24T00:00:01",
+            |						 "statement": "This is a statement",
+            |            "lateAppeal": false
+            |		}
+            |}
+            |""".stripMargin)
```

Comment:
This jsonToSubmit could be a def that is called:
`
def jsonToSubmit(technicalIssue: String): JsValue = Json.parse(
          s"""
            |{
            |    "sourceSystem": "MDTP",
            |    "taxRegime": "VAT",
            |    "customerReferenceNo": "123456789",
            |    "dateOfAppeal": "2020-01-01T00:00:00",
            |    "isLPP": false,
            |    "appealSubmittedBy": "customer",
            |    "appealInformation": {
            |					 	 "reasonableExcuse": $technicalIssue,
`